package.json

```diff
@@ -140,7 +140,7 @@
     "stylelint": "^13.0.0",
     "stylelint-config-sass-guidelines": "^7.0.0",
     "ts-loader": "^7.0.5",
-    "ts-pattern": "^4.2.2",
+    "ts-pattern": "4.0.5",
```

Comment:
This fixed it?

Reply:
Nahh. I could get more understandable error with this version(4.0.5) since ts-pattern's version and ours were nor compatible. But still getting the error unfortunately. 

Now I'm thinking 
`nodeLinker: node-modules <-> pnpm (or pnp)` impacted by rollup config. Probably we just need to use node-modules 😅

Reply:
Let me check more when I have spare time.

Reply:
yeah, is not big priority